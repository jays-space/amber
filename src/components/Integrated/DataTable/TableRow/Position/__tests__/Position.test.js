import { render, screen } from "@testing-library/react";
import "@testing-library/jest-dom";
import { Position } from "..";

describe("<Position />", () => {
  const mockPositionData = {
    role: "fullstack developer",
    team: "development team",
  };

  it("should render correctly", () => {
    render(<Position position={mockPositionData} />);

    const positionComponent = screen.getByTitle("position-applied");
    expect(positionComponent).toBeInTheDocument();
    expect(positionComponent).toMatchSnapshot();
  });
});
