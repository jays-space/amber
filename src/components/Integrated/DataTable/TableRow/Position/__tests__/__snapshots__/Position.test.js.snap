// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Position /> should render correctly 1`] = `
<div
  title="position-applied"
>
  <span>
    <p
      class="tracking-wide  text-cyan-900 font-semibold w-10/12 text-lg capitalize whitespace-nowrap text-ellipsis overflow-hidden"
      data-testid="paragraph"
    >
      fullstack developer
    </p>
  </span>
  <span>
    <p
      class="tracking-wide  text-slate-700 capitalize opacity-75"
      data-testid="paragraph"
    >
      development team
    </p>
  </span>
</div>
`;
